### ISSUE TITLE
A11y_AzureDataStudio_Deploymenttemplateconfiguration_ScreenReader:NVDA is not announcing disabled button as disabled.

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7677]
GitHubTags: #A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#DesktopApp;#Win32;#NVDA;
**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
**Repro Steps:**
1.Install /Launch the application Azure data studio.
2.Navigate to connections and click on more action button.
3.Select “Deploy SQL Server big data cluster” from more action drop down.
4.Click on “Deploy SQL Server big data cluster”.
5. "Select the deployment option" screen gets open and click on I accept check box and click on select button.
6. Start NVDA and navigate with the help of up and down arrow and observe the announcement.
**Actual:**
NVDA is announcing disabled button as normal enabled button.
**Expected:**
NVDA should announce disabled button as disabled button.
**User Impact:**
Screen reader User will misunderstood content and they will think there is some interactive button which can be used for some purpose.

**Attachment for Reference:**
[7677_A11y_AzureDataStudio_Selectthedeploymentoption_ScreenReader4.pptx](https://github.com/microsoft/azuredatastudio/files/3717655/7677_A11y_AzureDataStudio_Selectthedeploymentoption_ScreenReader4.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
we have removed the checkbox(emoji) in the latest design, see below, closing the issue.
<img width="600" alt="Screen Shot 2019-11-22 at 4 14 46 PM" src="https://user-images.githubusercontent.com/13777222/69469038-98751000-0d43-11ea-8646-3d54b1a34e6e.png">

**Comment by V-sheman** [comment]
Verified the issue in Version: 1.15.0-insider (user setup)
Commit: 56695be14ac0160e7a4d23b7af939b98fdaaea45
Date: 2020-01-30T05:35:45.707Z
VS Code: 1.42.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is obsoleted hence closing the issue. 

![image](https://user-images.githubusercontent.com/56344899/73432804-7f06ca80-4369-11ea-9e79-323fa90676c3.png)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-May2019;
